packages/js/src/structured-data-blocks/faq/components/Question.js

```diff
@@ -336,16 +336,18 @@ export default class Question extends Component {
 		return (
 			<div className="schema-faq-section" key={ id }>
 				<RichText
+					identifier={ id + "-question" }
```

Comment:
The `identifier` is the `id` + "-question" for a question and `id` + "-answer" for an answer. It's important to have a different `identifier` value for question and answer. Otherwise, the cursor within the block will be messed up when editing.

Reply:
... as we awkwardly found out during the refinement on Monday 😅